src/App.tsx

```diff
@@ -1,61 +1,34 @@
+import React, { useState } from 'react';
+
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList';
+import { Todo } from './types';
+import { TodoForm } from './components/TodoForm/TodoForm';
+import { getUserById } from './utils/getUserById';
+
+export const App: React.FC = () => {
+  const preparedTodos: Todo[] = todosFromServer.map(todo => {
+    return {
+      ...todo,
+      user: getUserById(todo.userId) || null,
+    };
+  });
+
+  const [todos, setTodos] = useState<Todo[]>(preparedTodos);
+
+  const handleAddTodo = (newElementTodo: Todo) => {
+    setTodos(curentTodos => [...curentTodos, newElementTodo]);
```

Comment:
[CODE STYLE] - Don't use one-letter variable naming. Instead of `e` in `setTodos(e => [...e, newElementTodo])` use a more descriptive name, for example `currentTodos`. This will make your code more readable and maintainable.